### PR TITLE
Refactoring UI for uniformity

### LINKED ISSUE #12: Refactoring for UI uniformity
The Home, All Courses, and About course pages have different UI schemes, making the UI inconsistent.
Some issues caught the eye too.
Examples follow

@abhishek97

### PR DESCRIPTION
**Pull request by VibhorCodecianGupta**
Resolves #12 
@abhishek97 please review

### CODE REVIEW

**Comment by VibhorCodecianGupta** [comment]
**_Home page_**

screen shot 2018-08-09 at 10 15 23 pm
screen shot 2018-08-09 at 10 15 43 pm
screen shot 2018-08-09 at 10 15 52 pm


The `All Courses` section on this page is deliberately not showing the number of available runs because there is a separate All Courses tab for the same, where users will get a more detailed layout, hence listing runs everywhere is redundant.

**Comment by VibhorCodecianGupta** [comment]
**_All Courses page_**

screen shot 2018-08-09 at 10 17 17 pm

screen shot 2018-08-09 at 10 16 38 pm

**Comment by VibhorCodecianGupta** [comment]
_**Course id Page**_

screen shot 2018-08-09 at 10 17 53 pm
screen shot 2018-08-09 at 10 18 31 pm
screen shot 2018-08-09 at 10 18 43 pm
screen shot 2018-08-09 at 10 18 53 pm


Redesigned course id page. 
Again, the recommended courses section doesn't list all the runs for similar reasons as stated above

File: views/index.hbs [hunk 1]
```diff
@@ -1,88 +1,97 @@
...
+                      </h2>
+                      <p class="card-text card-body">
+                          {{summary}}
+                      </p>
+                  </div>
+                  <div class="row no-gutters align-items-center">
+                      <div class="card-mentor col-12 col-sm-8 col-md-12">
+                          <div class="card-mentor-image">
+                            {{#each course.instructors as |instructor index|}}
+                              {{#ifCond index '<' 2}}
```
Comment (abhishek97) [inline views/index.hbs]: This way we will "lose" information if we have more than 2 mentors in a course. The primary aim for this project is to allow search engines to crawl the site for information. We need to figure out a way to show all of the information available.

Comment (VibhorCodecianGupta) [reply views/index.hbs]: If I'm not wrong, its just the picture that wont show right? the text would all still be there. Will that affect the SEO negatively to an extent we can't afford? If yes, any advice on the fix?

Comment (VibhorCodecianGupta) [reply views/index.hbs]: @abhishek97 I've pushed a temporary fix. I noticed there weren't more than 3/4 mentors and 3 mentor images for any course, so I brute forced the if condition to 3. Also changed the rupee symbol. 
please review

**Comment by VibhorCodecianGupta** [comment]
@abhishek97 updates?

**Comment by VibhorCodecianGupta** [comment]
@abhishek97 any updates?

**Pull request merged**